app/agents_v2/primary_agent/troubleshooting/workflow_library.py

```
@@ -0,0 +1,1013 @@
+"""
+Agent Sparrow - Troubleshooting Workflow Library
+
+This module implements a comprehensive library of structured troubleshooting workflows
+with progressive complexity handling and adaptive workflow selection.
+"""
+
+import logging
+from typing import Dict, List, Optional, Any
```

Ruff (0.11.9)

`typing.Any` imported but unused

Remove unused import: `typing.Any`

(F401)